### ISSUE TITLE
Training with 'all'

### ISSUE THREAD

**Issue by dagap** [MIC-DKFZ/nnUNet#499]
This is more of a question. As I understand it, training with 'all' will use the whole training dataset and no CV folds. In such a case, is there still a training/validation split during the training? If so, what is the ratio of this split and how does it differ from a single fold?

**Comment by FabianIsensee** [comment]
If you train with all training cases then there is no validation set. If you would like to have a single train:val split you should just train a single fold. That is then going to be 80:20

**Comment by dagap** [comment]
Ah right, ok. Cool, thanks for the quick clarification. You are awesome. 
Is there a presentation about nn-Unet somewhere or just some slides from a conference or something?

**Comment by FabianIsensee** [comment]
Unfortunately no, but there should be plenty of information in the Nature Methods paper. If something is missing, the answer is in the code. Or you ask a question in the Discussions
